package.json

```diff
@@ -36,8 +36,9 @@
     "access": "public"
   },
   "dependencies": {
-    "@rollup/plugin-node-resolve": "^15",
+    "@brightspace-ui/intl": "^3",
```

Comment:
Does this mean we can't use `@brightspace-ui/testing` in `intl`, or would that still work? 🤔 Looks like we use mocha directly anyways so maybe doesn't matter